ddl.dic

```diff
@@ -772,11 +772,12 @@ save_dictionary_valid.attributes
 
     _definition.id                '_dictionary_valid.attributes'
     _definition.class             Attribute
-    _definition.update            2013-03-06
+    _definition.update            2021-07-21
     _description.text
 ;
     A list of the attribute names and categories that are assessed
-    for application in the item, category and dictionary definitions.
+    for application in the item, category and dictionary definitions. A
+    parent attribute category implicitly includes all child categories.
```

Comment:
I propose that the definition be further improved by explicitly stating that the inclusion of child categories is recursive, that is, that it also includes the (grand)+child categories. I.e. `A parent attribute category implicitly includes all child categories in a recursive manner.` or something similar, but better sounding.

Reply:
Great, thank you for the revision.